src/Components/BookPage/BookSynopsis.js

```diff
@@ -1,65 +1,467 @@
+            }
+          </Box>
+        </CardContent>
+      </Card>
+      <div>
+        <Card sx={styles.sideCard}>
+          <CardContent>
+            <Box sx={styles.sideCardContainer}>
+              <Typography sx={styles.sideCardTitle}>About This Book</Typography>
+              <Box sx={styles.sideCardLinkContainer}>
+                <Typography sx={styles.creator}> Written by: </Typography>
+                {' '}
+                <Link style={styles.link} to={`/creator/${authorID}`}>
+                  {authorName}
+                </Link>
+                <br />
+                <Typography sx={styles.creator}> Illustrated by: </Typography>
+                {' '}
+                <Link style={styles.link} to={`/creator/${illustratorID}`}>
+                  {illustratorName}
+                </Link>
+              </Box>
+              <div style={styles.blockContainer}>
+                <div style={styles.block}>
+                  <Typography sx={styles.bolded}>
+                    {ageRange}
+                  </Typography>
+                  <p style={styles.sub}>Age Range</p>
+                </div>
+                {
+                    (gradeMin || gradeMax) ? (
+                      <div style={styles.block}>
+                        <Box sx={styles.bolded}>
+                          {gradeRange}
+                          {' '}
+                        </Box>
+                        <p style={styles.sub}>Grade level</p>
+                      </div>
+                    ) : <div />
+                  }
+              </div>
+              {
+                  (bookshopURL) ? (
+                    <Button
+                      sx={styles.bookshop}
+                      endIcon={<Away />}
+                      size="large"
+                      href={bookshopURL}
+                      target="_blank"
+                    >
+                      Buy from Bookshop.org
+                    </Button>
+                  ) : <div />
+                }
+            </Box>
+          </CardContent>
+        </Card>
+        <Card sx={styles.sideCard}>
+          <CardContent>
+            <Box sx={styles.sideCardContainer}>
+              <Typography sx={styles.sideCardTitle}>Additional Resources</Typography>
+              <Box sx={styles.sideCardLinkContainer}>
+                { (readAloudURL) ? (
+                  <div>
+                    <LinkUI sx={styles.linkUI} href={readAloudURL} rel="noreferrer" target="_blank">
+                      Story Read Aloud
+                    </LinkUI>
+                  </div>
+                ) : <div />}
+                { (bookshopURL)
+                  ? (
+                    <div>
+                      <LinkUI sx={styles.linkUI} href={bookshopURL} rel="noreferrer" target="_blank">
+                        Educator Guide
+                      </LinkUI>
+                    </div>
+                  ) : <div />}
+              </Box>
```

Comment:
It seems like you aren't using the educator links at all, let's talk with the designers about how to handle them!